pkg/v1/tkg/clusterclient/clusterclient.go

```diff
@@ -2338,6 +2340,18 @@ func (c *client) ListCLIPluginResources() ([]cliv1alpha1.CLIPlugin, error) {
 	return cliPlugins.Items, nil
 }
 
+// IsClassyCluster check whether cluster is ClusterClass based or not
+func (c *client) IsClassyCluster(clusterName, namespace string) (bool, error) {
```

Comment:
Is there a unit test for this?

Reply:
Good catch. Added unit tests for this function.